adb_graphics/datahandler/gribfile.py

```diff
@@ -141,6 +139,7 @@ def _get_grid_suffix(filenames):
         for files in filenames.values():
             if files:
                 gfile = xr.open_dataset(files[0],
+                                        cache=False,
```

Comment:
If we turn off caching here, we may see less memory usage as we reference more and more fields. The docs say this about cached=True, the defualt:

> If True, cache data loaded from the underlying datastore in memory as NumPy arrays when accessed to avoid reading from the underlying data- store multiple times.